zilliqa-macros/src/test.rs

```diff
@@ -56,14 +61,18 @@ pub(crate) fn test_macro(_args: TokenStream, item: TokenStream) -> TokenStream {
                     let network = crate::Network::new(std::sync::Arc::new(std::sync::Mutex::new(rng)), 4, seed);
 
                     // Call the original test function, wrapped in `catch_unwind` so we can detect the panic.
-                    let result = futures::FutureExt::catch_unwind(std::panic::AssertUnwindSafe(#inner_name(network))).await;
+                    let span = tracing::span!(tracing::Level::INFO, "tst");
```

Comment:
Useful if you are finding that your test only fails when you are running multiple at once (such as when time is interfering). Can add the seed or task id to the span when debugging.